sajibportfolio.html

```diff
@@ -1,273 +1,321 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="/sajibportfolio.css" />
 
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" type="text/css" href="sajibportfolio.css" />
-  <title>Document</title>
-</head>
```

Comment:
- [ ] Dear @sajibarafatsiddiqui, I see you updated your styles. This is great the only problem is that you have introduced a bug to your project. You use `href="/sajibportfolio.css"` to refer to your CSS. this is causing an issue as it's not mentioned to the browser you are using a relative address. Please instead use  `href="./sajibportfolio.css"`. the `./` tells the browser you are referring to the relative address.
> Note: this applies to all other `href` or `src` you used. Always favor `./` over `/`.

Reply:
resolved .